source/templates.tex

```diff
@@ -277,17 +277,16 @@
 \tcode{typename}
 followed by a
 \grammarterm{qualified-id}
-denotes the type in a non-type%
+denotes the type in a \grammarterm{parameter-declaration}.%
 \footnote{Since template
 \grammarterm{template-parameter}{s}
 and template
 \grammarterm{template-argument}{s}
 are treated as types for descriptive purposes, the terms
-\term{non-type parameter}
-and
-\term{non-type argument}
-are used to refer to non-type, non-template parameters and arguments.}
-\grammarterm{parameter-declaration}.
+non-type \grammarterm{template-parameter} and
+non-type \grammarterm{template-argument}
```

Comment:
I actually think those definitions should be normative and indexed; they are used quite a lot and we shouldn't rely on the "since" rationale offered here for readers to understand a fairly basic term.

Reply:
Should we do _non-type template-parameter_ like we do for _trailing requires-clause_, or _non-type template parameter_? The former seems to be used more often than the latter.